src/Tizen.NUI/src/public/BaseComponents/View.cs

```diff
@@ -152,6 +152,11 @@ internal View(global::System.IntPtr cPtr, bool cMemoryOwn, bool shown = true) :
                 SetVisible(false);
             }
 
+            if (DefaultGrabTouchAfterLeave == true)
+            {
+                GrabTouchAfterLeave = true;
+            }
```

Comment:
`GrabTouchAfterLeave`의 기본값이 현재는 false라서 상관은 없지만, 
되게 애매한 코드입니다.

```suggestion
            GrabTouchAfterLeave = DefaultGrabTouchAfterLeave;
```

Reply:
네 false임에도 불구하고 매번 프로퍼티를 set해야 하나라는 생각으로 저렇게 코드를 넣긴 했는데
이상해 보이네요 수정하겠습니다.